graphql/schemabuilder/pagination.go

```diff
@@ -664,7 +678,7 @@ func (c *connectionContext) getConnection(ctx context.Context, out []reflect.Val
 
 	// If the pagination is externally managed, thunder isn't going to handle setting page
 	// information or reducing the edges.
-	if c.IsExternallyManaged() {
+	if c.IsExternallyManaged() && !c.PostProcessOptions.ApplyTextFilter {
```

Comment:
I imagine if we add more postprocessing options that could affect pagination info in the future we'd have to remember to update this `if` statement.  Maybe it would be more maintainable to have this be calculated in a method on `PostProcessOptions`?  Imagining something like:
```
func (p *PostProcessOptions) SetPaginationInfo() bool {
    return p.ApplyTextFilter
}
```

Reply:
Agreed. Going further, however, I would also decouple the ApplyTextFilter with SetPaginationInfo logic. With the above approach, resolvers that will want to set a paginationInfo without having to apply a text filter will be forced to also apply the text filter.